internal/zinc-scripted/src/main/contraband/scripted.contra

```diff
@@ -0,0 +1,14 @@
+package sbt.internal.inctest
```

Comment:
Ok so you need to deserialise some JSON. Don't need contraband or .contra for that.